### PR TITLE
Resolução dos exercícios 1.i, 1.ii e 1.iii

### PR DESCRIPTION
**Pull request by LydiaBraga**
Resolução dos exercícios 1.i, 1.ii e 1.iii da disciplina DM104.

### CODE REVIEW

**Comment by edysegura** [comment]
Olá @LydiaBraga !

Você prefere receber os meus comentários em Inglês ou Português?

**Comment by LydiaBraga** [comment]
Oi @edysegura!
Pode ficar a vontade para comentar em inglês, se quiser :)

File: lydia-braga/1.i/si-page.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (edysegura) [inline lydia-braga/1.i/si-page.html]: As this document is in portuguese language we need to update the attribute `lang` to "pt-br"

File: lydia-braga/1.ii/cv.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (edysegura) [inline lydia-braga/1.ii/cv.html]: As this document is in portuguese language we need to update the attribute `lang` to "pt-br"

File: lydia-braga/1.iii/livro.html [hunk 3]
```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (edysegura) [inline lydia-braga/1.iii/livro.html]: As this document is in portuguese language we need to update the attribute `lang` to "pt-br"

File: lydia-braga/1.ii/cv.html [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+    </section>
+
+    <section>
+        <h2>Experiência</h2>
+
+        <h3>Especialista em Sistemas</h3>
+        <small>
+            Inatel<br>
+            Janeiro de 2016 - Presente
+        </small>
```
Comment (edysegura) [inline lydia-braga/1.ii/cv.html:35]: Why did you choose to use `<small>` here?

File: lydia-braga/1.iii/livro.html [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Harry Potter e a Pedra Filosofal</title>
```
Comment (edysegura) [inline lydia-braga/1.iii/livro.html:7]: Are you a big Harry Potter fan? ;)

File: lydia-braga/1.ii/cv.html [hunk 6]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Currículo</title>
```
Comment (edysegura) [inline lydia-braga/1.ii/cv.html]: Here would be nice to have your name instead of only "Currículo"

Comment (edysegura) [inline lydia-braga/1.ii/cv.html]: Here would be nice to have your name instead of only "Currículo"

**Review by edysegura — COMMENTED** [review]
Hi @LydiaBraga 

If you have any doubts about my comments please let me know.

Cheers,

File: lydia-braga/1.ii/cv.html (hunk 4 above)
Comment (LydiaBraga) [reply lydia-braga/1.ii/cv.html:35]: This tag was really difficult to choose. I tried many others until I got to this one.
The goal of this text is to be a small detail under the title, so it is not a subtitle, neither a quotation or a paragraph.
Looking at the mdn documentation, I found out that small is used to represent side-comments and small print, so it was the closest I could get to what I wanted to represent.

Do you know another tag that fits better for this situation?

File: lydia-braga/1.iii/livro.html (hunk 5 above)
Comment (LydiaBraga) [reply lydia-braga/1.iii/livro.html:7]: hahahah!
Yes, I am! :D

**Comment by LydiaBraga** [comment]
Hi @edysegura!
All my corrections were done.
Waiting for your avaliation!

Thanks!

**Pull request merged**